app/models/article.rb

```diff
@@ -0,0 +1,29 @@
+class Article < ActiveRecord::Base
+
+	def self.create_articles(articles_data)
+		articles_data.each do |article|
+      article = _build_object(article)
+			Article.create(
+				title: article.title,
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.